testbot.py

```diff
@@ -160,4 +165,46 @@ async def join_game(arguments: list, message: discord.message):
     logging.info(f'Assigned {message.author} to {game_name}')
     await message.channel.send(f'Have fun playing {game_name}, {message.author}!')
 
+
+async def make_poll(message: discord.message):
+    command = message.content.replace("pb poll", '').strip()
+
+    if not command or "\"" not in command or ',' not in command:
+        logging.debug('No poll title or arguments were passed')
+        await message.channel.send('You can\'t just make a poll without a title or arguments! Please insert a poll '
+                                   'title that is surrounded in \"\" as well as arguments proceeded with commas. In '
+                                   'addition, the minimum amount of arguments is 2!')
+        return
+
+    title = command[command.find("\"") + 1:command.rfind("\"")]
+    options = command.replace(f'\"{title}\"', '').strip().split(", ")
+    logging.info(command)
+    arguments = {
+        "title": title,
+        "options": options
+    }
+
+    options = arguments.get("options")
+    argument_length = len(options)
+    if argument_length > 8:
+        logging.debug('Too many arguments passed')
+        await message.channel.send("Too many arguments passed! The maximum amount of arguments you can add is 8!")
```

Comment:
The user doesn't know what an argument is; they passed a title and comma separated options!